src/app/core/components/content-top-bar/content-top-bar.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component, Input } from '@angular/core';
+import { ModeAction, ModeService } from '../../../shared/services/mode.service';
+import { ContentInfo } from '../../../shared/models/content/content-info';
+
+@Component({
+  selector: 'alg-content-top-bar',
+  templateUrl: './content-top-bar.component.html',
+  styleUrls: [ './content-top-bar.component.scss' ],
+})
+export class ContentTopBarComponent {
+  @Input() fullFrameContent?: boolean;
+  @Input() currentMode?: string;
```

Comment:
About `currentMode` type, shouldn't it be `Mode` like in `TopBarComponent` ?

Reply:
Fixed